pkg/sql/convert/create.go

```diff
@@ -1,11 +1,10 @@
 package convert
 
 import (
-	"context"
 	"fmt"
 	"github.com/lib/pq"
 	"github.com/nrc-no/core/pkg/api/types"
-	"github.com/nrc-no/core/pkg/sql/schema"
+	sqlschema2 "github.com/nrc-no/core/pkg/sql/schema"
```

Comment:
```suggestion
	"github.com/nrc-no/core/pkg/sql/schema"
```
... and rename the rest